code/modules/antagonists/heretic/status_effects/buffs.dm

```diff
@@ -161,7 +191,8 @@
 	if(QDELETED(src) || QDELETED(owner))
 		return
 
-	var/obj/effect/floating_blade/blade = new blade_type(get_turf(owner))
+	var/obj/effect/floating_blade/blade
+	blade = new blade_type(get_turf(owner))
```

Comment:
Why this change?

Reply:
Fixes a runtime causing the ascension to not spawn the ring of knives.